aptos-move/block-executor/src/executor.rs

```diff
@@ -240,7 +240,12 @@ where
             },
             ExecutionStatus::Abort(err) => {
```

Comment:
where is regular transaction abort and commit onchain handled? is that in ExecutionStatus::Success ?

Reply:
Yes, as far as I know even that sort of failure is part of Success status.